Adds the Disciplinary Board Room

Hubblenaut opened the pull request:
:cl:
rscadd: Adds the Disciplinary Board Room.
/:cl:

The result of two hours of tinkering with others on Discord:
![Image](https://i.imgur.com/7p87I5D.png)
Intended primarily for court martials, in the hope to facilitate the organisation and execution of one and to motivate players to have them.

There is intended space for the three presiding officers, the accused, the record keeper, an impartial investigator and a guard near the door.
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

mustafakalash commented:
Is the fad of public court martials over?

mustafakalash commented:
Also, tableflag is cute 10/10.

HeyBanditoz commented:
I like it, but how much is it going to be used?

Hubblenaut commented:
If it simply makes players inclined to organise court martials a bit more often, my goal here is already achieved. It's also nice to have for those players that already try to regularly have them.

chaoko99 commented:
is there a firelock here? Also, have you moved the closets and whatnot that were there before you added it?

Hubblenaut commented:
Everything in maintenance was merely shifted around. There has been a large gap in maintenance ever since I removed the relay room.

Also, added missing intercoms.

Sbotkin commented:
Looks _fabulous_ but remove the reinforced walls, it's not a secure area. Maybe even replace them with tintable windows, with the button behind the judges (?) chairs.

>\## UNIT_TEST \##: [AREA: Areas shall be coherent]: Bridge Aft Starboard Maintenance (/area/maintenance/bridge/aftstarboard) ([0x40000e2]) (77,105,5) is incoherent. Expected 1 subarea, fill gave 2.

Also, use pixel_[xy], not step_[xy].

Rebruiza commented:
Fairly sure a court martial/"temporary prison" (there's a literal prisoner/incarcerated person in there when in use) deserves reinforced walls.

Hubblenaut committed:
Adds the Disciplinary Board Room

Hubblenaut commented:
Don't know how I always end up accidentally editing `step_`. Also, travis error fixed.

Sbotkin commented:
It's more of an audition room, not a prison nor for bigger criminals.

Hubblenaut commented:
FYI I chose reinforced walls as every single office on that deck is fully surrounded by it, and I saw no reason a court room would be protected any less than those. However the maintenance door exists majorly to keep antagonists in mind.

Rebruiza commented:
Maintenance doors exist to allow maintenance, such as Engineering, Janitorial, and so on.

Rebruiza commented:
Maintenance doors exist to allow maintenance, such as Engineering, Janitorial, and so on. As they are access points, they are as much of a door to a janitor as to a raider.

Hubblenaut commented:
>Maintenance doors exist to allow maintenance, such as Engineering, Janitorial, and so on. As they are access points, they are as much of a door to a janitor as to a raider.

?? I just explained why I mapped in a maintenance door. I'm not sure what you're trying to say.

Rebruiza commented:
>However the maintenance door exists majorly to keep antagonists in mind.

And I explained that the maintenance doors primary function is actually crew, and not antagonists.

Hubblenaut commented:
>And I explained that the maintenance doors primary function is actually crew, and not antagonists.

I mapped it majorly for antagonists. It could have just as well been a wall. I really don't understand what you're trying to say. Nobody is discussing what maintenance doors are generally good for.
